packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -566,6 +566,80 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     }).then<int>((_) => count);
   }
 
+  /// Triggers multiple frames for `duration` amount of time or until there are
+  /// no longer any frames scheduled, whichever is earlier.
+  ///
+  /// This does not promise settle of the widget, and there maybe remaining
+  /// animation and therefore undisposed `Ticker`s.
+  ///
+  /// This will call [pump] at least once, even if no frames are scheduled when
+  /// the function is called, to flush any pending microtasks which may
+  /// themselves schedule a frame.
+  ///
+  /// The `frameRefreshRate` specify the frequency (in Hz) for frame requests.
+  /// This is an analog to screen refresh rate on a device. The frequency rate
+  /// will be rounded down to integer microseconds frame interval.
+  /// Default to be 59.94 Hz.
```

Comment:
This isn't a complete sentence - it would be helpful to say why this is the default as well.